Healthy/Classes/Generated/UIImage.Generated.swift

```diff
@@ -9,4 +9,12 @@ extension UIImage {
     static var accentColor: UIImage {
       UIImage(named: "AccentColor")!
     }
+
+    static var facebookIcon: UIImage {
+      UIImage(named: "facebook_icon")!
+    }
+
+    static var googleIcon: UIImage {
+      UIImage(named: "google_icon")!
```

Comment:
```suggestion
      UIImage(named: "icon-facebook")!
    }

    static var googleIcon: UIImage {
      UIImage(named: "icon-google")!
```

Please rename the assets to match the given name.